pkg/client/restclient/client.go

```diff
@@ -87,7 +87,7 @@ func NewRESTClient(baseURL *url.URL, versionedAPIPath string, config ContentConf
 		config.GroupVersion = &unversioned.GroupVersion{}
 	}
 	if len(config.ContentType) == 0 {
-		config.ContentType = "application/json"
+		config.ContentType = "application/vnd.kubernetes.protobuf"
```

Comment:
We may have to hoist this to just controller-manager, kubelet, and kube-proxy initialization.